invenio/modules/tags/models.py

```diff
@@ -257,7 +257,8 @@ def __init__(self, bibrec=None, **kwargs):
         """TODO."""
         super(WtgTAGRecord, self).__init__(**kwargs)
 
-        self.bibrec = bibrec
+        if bibrec is not None:
```

Comment:
> Do you have any idea why is implemented the `__init__`?

Maybe the author wanted to use something like `WtgTAGRecord(bibrec=Record.query.get(recid))`, but I'm not sure ...